src/bosh-director/spec/unit/jobs/cleanup_artifacts_spec.rb

```diff
@@ -3,6 +3,12 @@
 
 module Bosh::Director
   describe Jobs::CleanupArtifacts do
+    let(:event_manager) {Api::EventManager.new(true)}
```

Comment:
One of these days I'll go through and reformat all the `let`s in our specs to consistently use spacing inside the curly brackets...